CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 #### Features
 -- Added `--show-fullpath` flag to `ls`. (#[596](https://github.com/peak/s5cmd/issues/596))
 #### Improvements
+--- Implemented concurrent multipart download support for `cat`. ([#593](https://github.com/peak/s5cmd/pull/593)) 
```

Comment:
Please use a single dash for bullet points;
![image](https://github.com/peak/s5cmd/assets/7118751/b1c3ef44-c170-4dbb-b164-b549b0b28256)
https://github.github.com/gfm/#bullet-list-marker

Reply:
The line above has the same problem, would you like me to fix it in this PR or make another one ?

Reply:
you may change it in this pr since it's a trivial change.